### ISSUE TITLE
S3 / Active Storage

### ISSUE THREAD

**Issue by jjkoehorst** [seek4science/seek#1113]
With the ever increasing flow of information I was wondering if there are any thoughts on using / supporting Active Storage in SEEK? 

For example using S3 to enable the storage of seek data into a remote/local storage?

See for example https://edgeguides.rubyonrails.org/active_storage_overview.html

**Comment by stuzart** [comment]
Yes for a long time I've been keen to move to use Active Storage, but the recent addition of the git support and reliance on a standard file system and directories makes this more difficult. But it is something I'd like to come back to and revisit. I was also looking at it again recently for how we handle images and resizing
